Tests/HyphenationTests/ParagraphHyphenationServiceTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\HyphenationTests;
+
+use App\Services\HyphenationService;
+use App\Services\ParagraphHyphenationService;
+use PHPUnit\Framework\TestCase;
+
+class ParagraphHyphenationServiceTest extends TestCase
+{
+    public function testGetSyllables(): void
+    {
+        $hyphenationServiceMock = $this->createMock(HyphenationService::class);
+
+        $hyphenationServiceMock
+            ->method('getSyllables')
+            ->willReturn(['.ach4', '.ad4der', '.af1t', '.al3t', '.am5at']);
+
+        $paragraphHyphenationService = new ParagraphHyphenationService($hyphenationServiceMock);
+
+        $syllables = $paragraphHyphenationService->getSyllables();
```

Comment:
Well this is not exactly good usage of mock, as main question is what are you testing here? From current code I think you are testing if mocking works 😄 

Mock should be used to provide information to a service, with which the service would work. 

So I will give simple example for a guidance.

Lets say we have a hyphenation service that takes syllables from API. We wish to mock response from that API as the service will not use API on tests (that would be silly). Now as we have a mock response we can test if the service hyphenates word correctly. 

We can discus this more f2f.